symphony/lib/toolkit/class.datasource.php

```diff
@@ -241,6 +241,45 @@ public function emptyXMLSet(XMLElement $xml = null)
     }
 
     /**
+     * Construct an XML fragment that describes a pagination structure.
+     * Do not call as a static function, since this is only temporary.
```

Comment:
It SHOULD be static, but it can't right now (it was crashing on my end...) Maybe I need to revisit this.